core/src/main/java/com/netflix/conductor/core/execution/DeciderService.java

```diff
@@ -283,7 +284,10 @@ void updateWorkflowOutput(final Workflow workflow, @Nullable Task task) {
             output = externalPayloadStorageUtils.downloadPayload(last.getExternalOutputPayloadStoragePath());
             Monitors.recordExternalPayloadStorageUsage(last.getTaskDefName(), ExternalPayloadStorage.Operation.READ.toString(), ExternalPayloadStorage.PayloadType.TASK_OUTPUT.toString());
         } else {
-            output = last.getOutputData();
+            Map<String, Status> executedTasks = new HashMap<>();
+            allTasks.forEach(_task -> executedTasks.put(_task.getReferenceTaskName(), _task.getStatus()));
+            Map<String, Object> outputObj = ImmutableMap.of("task", last.getReferenceTaskName(), "status", last.getStatus(), "output", last.getOutputData(), "executedTasks", executedTasks.toString());
+            output = outputObj;
```

Comment:
Hey @X-Ultra, We don't see the value in adding all executedTasks status to workflow output. This is readily available from getWorkflow API anyways, if anyone if interested.